compiler/src/dotty/tools/dotc/transform/PostTyper.scala

```diff
@@ -185,6 +193,16 @@ class PostTyper extends MacroTransform with IdentityDenotTransformer  { thisTran
           if (ctx.owner.enclosingMethod.isInlineMethod)
             ctx.error(SuperCallsNotAllowedInline(ctx.owner), tree.pos)
           super.transform(tree)
+        case tree: Apply =>
+          methPart(tree) match {
```

Comment:
Bugfix. I believe we did not check new's for instantiatability if the type was parameterized. The problem is that the type in the new itself is only the type constructor, which can have good bounds even though the parameter instantiation has not.